### ISSUE TITLE
Deposit calculated values for provided files when available

### ISSUE THREAD

**Issue by davidlmobley** [MobleyLab/benchmarksets#53]
To move this in the direction of helping people benchmark, we should provide calculated values from gold standard calculations with the provided files, when available. These should be in a markdown file in the relevant directory, I think.

@nhenriksen - is this something you're able to add? I think you have values for all of the files you've deposited? 

@GHeinzelmann - I think you may not? 

@Janeyin600 - do you? 

At some point we'll actually need to repeat the lysozyme calculations (or another group will) and get input files for those, and calculated values, in here as well.

**Comment by jchodera** [comment]
@andrrizzi and @lnaden are actively working to use YANK to repeat host-guest and lysozyme calculations right now.

**Comment by GHeinzelmann** [comment]
I do have all the files from my BRD4 calculations stored, I can organize them and add to the input files if needed.

**Comment by nhenriksen** [comment]
@davidlmobley yes, I can add these!

**Comment by davidlmobley** [comment]
@jchodera 

> @andrrizzi and @Lnaden are actively working to use YANK to repeat host-guest and lysozyme calculations right now.

Yes, that's where I'm hoping the results and files will come from!

**Comment by ghost** [comment]
@davidlmobley  I can add those
